### ISSUE TITLE
aria-label missing for the share buttons

### ISSUE THREAD

**Issue by josevarghese** [superwebshare/super-web-share#3]
The aria-label is missing within the share button. We need to add it for screen readers or for Accessibility.

* [x] I've read and understood the [contribution guidelines](https://github.com/superwebshare/SuperWebShare-WordPress-Plugin/blob/master/.github/CONTRIBUTING.md).
* [x] I've searched for any related issues and avoided creating a duplicate issue.

**Comment by josevarghese** [comment]
Issue fixed by the commit of #6
